backend/src/core/models/user/tests/user.test.ts

```diff
@@ -0,0 +1,110 @@
+import { Sequelize } from 'sequelize-typescript'
+import sequelizeLoader from '@test-utils/sequelize-loader'
+import { User, Domain, Agency } from '@core/models'
+import { validateDomain } from '@core/utils/validate-domain'
+import { RedisService } from '@core/services'
+import config from '@core/config'
+
+let sequelize: Sequelize
+
+jest.mock('@core/utils/validate-domain')
+const mockValidateDomain = validateDomain as jest.MockedFunction<
+  typeof validateDomain
+>
+
+beforeAll(async () => {
+  sequelize = await sequelizeLoader(process.env.JEST_WORKER_ID || '1')
+})
+
+afterEach(async () => {
+  await User.destroy({ where: {} })
+  await Domain.destroy({ where: {} })
+  await Agency.destroy({ where: {} })
+})
+
+afterAll(async () => {
+  await sequelize.close()
+  await RedisService.shutdown()
```

Comment:
Btw, the Redis clients aren't shutting down for me locally (though Travis still passes it)...
Not sure how to fix this

Reply:
Also, not sure if I've covered all the cases for this test